spiceypy/utils/support_types.py

```diff
@@ -66,122 +66,48 @@
 
 import numpy
 from numpy import ctypeslib as numpc
-import spiceypy.utils.exception_lookups as exception_lookups
-
-# Collection of supporting functions for wrapper functions
-__author__ = "AndrewAnnex"
-
-errorformat = """
-================================================================================
-
-Toolkit version: {tkvsn}
 
-{short} --
-{explain}
-{long}
+from .exceptions import (
+    SpiceyError,
+    SpiceyPyError,
+    NotFoundError,
+    SpiceyPyIOError,
+    SpiceyPyMemoryError,
+    SpiceyPyTypeError,
+    SpiceyPyKeyError,
+    SpiceyPyIndexError,
+    SpiceyPyRuntimeError,
+    SpiceyPyZeroDivisionError,
+    SpiceyPyValueError,
+    exceptions,
+)
 
-{traceback}
 
-================================================================================\
-"""
+# Collection of supporting functions for wrapper functions
+__author__ = "AndrewAnnex"
 
 
-class SpiceyError(Exception):
-    """
-    SpiceyError wraps CSPICE errors.
-    :type value: str
+def short_to_spiceypy_exception_class(short: str) -> Type[SpiceyError]:
```

Comment:
Ah, cool, another piece of Python notation I'd never seen before. Is the annotation needed to make it work?

Reply:
SpiceyPy as of 3.0.0 has switched to using type hints, this allows sphinx to autogenerate type information for the docs and for pycharm to do it's magic.

Reply:
Oh, how cool.